### ISSUE TITLE
Making bundle upload errors less cryptic

### ISSUE THREAD

**Issue by DaRealNim** [codalab/codabench#808]
When uploading a competition bundle on codabench and something goes wrong, some errors are detailed, but most of the time we get the really cryptic error:

![image](https://user-images.githubusercontent.com/43792539/232340568-7f734f97-e28b-40c4-9716-e2402e918379.png)

which makes it really hard to debug. Maybe simply including raw error messages when this kind of fatal error happens would be helpful to get a sense of what needs to be fixed with the bundle.

**Comment by DaRealNim** [comment]
I solved my issue, I realized I had put a "-" before the "tasks" field while writing a phase

![image](https://user-images.githubusercontent.com/43792539/232780087-d4ab7c26-15f3-4a8c-b348-ba9b2973e77b.png)

**Comment by Didayolo** [comment]
You are absolutely right. We should simply add the raw error log to the red error message. That is extremely important.

**Comment by Didayolo** [comment]
I close the issue as it will soon be fixed by #824
